GliaWidgets/Sources/QuickLookBased/QuickLookBased.Interface.swift

```diff
@@ -1,6 +1,4 @@
-import Foundation
```

Comment:
could you please check Integration test for SwiftPackageManager to be sure that these imports are not required for SwiftPackageManager integration?